raiden-dapp/src/components/AddressInput.vue

```diff
@@ -240,65 +249,46 @@ $dark_border: #323232;
 $dark_background: #323232;
 
 .address-input--invalid ::v-deep .v-messages {
-  border-color: $dark_border;
-  background-color: $dark_background;
-  border: 1px solid !important;
-  border-radius: 5px;
+  border: none !important;
 }
 
 .address-input--invalid ::v-deep .v-messages:after {
   content: ' ';
-  border: solid;
-  border-radius: 1px;
-  border-width: 0 1px 1px 0;
   position: absolute;
   left: 50%;
   bottom: 90%;
   display: inline-block;
   padding: 3px;
-  transform: rotate(-135deg);
-  -webkit-transform: rotate(-135deg);
-  border-color: $dark_border;
-  background-color: $dark_background;
-}
-
-.address-input--hint-visible ::v-deep .v-messages {
-  color: #696969 !important;
-  font-family: Roboto, sans-serif;
-  font-size: 16px;
-  line-height: 19px;
-  margin-top: 4px;
-  margin-bottom: 14px;
 }
 
 .address-input--hint-visible ::v-deep .v-text-field__details {
   padding-top: 0;
   margin-top: 0;
 }
 
-.address-input__status-icon__icon {
-  line-height: 20px;
-  width: 20px;
-}
-
-.address-input__status-icon {
-  padding: 8px;
+::v-deep .v-text-field__details {
+  height: 30px;
 }
 
 .address-input--untouched {
   caret-color: white !important;
   color: white !important;
 }
 
-::v-deep .v-input__icon--clear {
-  margin-top: 6px;
-}
-
 ::v-deep .v-text-field > .v-input__control > .v-input__slot::before {
-  border-width: 2px 0 0 0;
+  border-width: 0 0 0 0;
 }
 
 ::v-deep .v-text-field > .v-input__control > .v-input__slot::after {
-  border-width: 3px 0 0 0;
+  border-width: 0 0 0 0;
+}
+
+.address-input__status__paste-button {
+  display: none;
```

Comment:
Why is the paste button not displayed?

Reply:
@nephix because currently, it doesn't work but I didn't want to completely remove it, see comment above.